packages/frontend/build.gradle.kts

```diff
@@ -11,19 +12,31 @@ version = rootProject.version as String
 
 kotlin {
   explicitApi()
+
+  sourceSets {
+    val jsMain by getting {
+      dependencies {
+        implementation(kotlin("stdlib-js"))
```

Comment:
this will probably break sonar coverage reporting and other stuff that relies on source set paths